nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/IdentifyMimeType.java

```diff
@@ -124,7 +147,7 @@ public void process(final InputStream stream) throws IOException {
                     TikaInputStream tikaStream = TikaInputStream.get(in);
                     Metadata metadata = new Metadata();
                     // Add filename if it exists
```

Comment:
probably makes sense to remove this comment now given that whether it will add it is optional